lib/phoenix_storybook/templates/layout/root_iframe.html.heex

```diff
@@ -7,6 +7,7 @@
     <%= if path = storybook_js_path(@conn) do %>
       <script
         nonce={csp_nonce(@conn, :script)}
+        defer={storybook_js_type(@conn) == "module"}
```

Comment:
`defer` is the default for `type="module"` scripts, so this one shouldn't be needed.